src/components/IssueSearchAdvanced.vue

```diff
@@ -10,13 +9,14 @@ type Facet = {
   value: string;
 };
 
-const emit = defineEmits<{
-  'issues:load': [any[]];
+const props = defineProps<{
+  isLoading: boolean;
 }>();
 
+const emit = defineEmits(['set:filters']);
```

Comment:
We tend to use the type signature declaration style for emits to leverage typescript checks, declaring a parameter type for `set:filters` would be useful here.